src/stories/epics/createForm.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { action } from '@storybook/addon-actions';
+import { Story } from '@storybook/react/types-6-0';
+import React, { ComponentProps } from 'react';
+
+import CreateEpicModalComponent from '~js/components/epics/createForm';
+
+import { withRedux } from '../decorators';
+import { sampleProject1, sampleUser1 } from '../fixtures';
+
+export default {
+  title: 'Epics/CreateEpicModal/Component',
+  component: CreateEpicModalComponent,
+  decorators: [withRedux()],
+};
+
+const Template: Story<ComponentProps<typeof CreateEpicModalComponent>> = (
+  args,
+) => <CreateEpicModalComponent {...args} />;
+
+export const Component = Template.bind({});
+Component.args = {
```

Comment:
When there's only one component, if the name here (`Component`) matches the title above (`Epics/CreateEpicModal/Component`) then it renders a single storybook page instead of a folder with only one page in it.